app/services/participant_validation_service.rb

```diff
@@ -19,75 +19,60 @@ def validate
     validated_record = matching_record(trn: trn, nino: nino, full_name: full_name, dob: date_of_birth)
     return if validated_record.nil?
 
-    validation_data = {
-      trn: validated_record[:teacher_reference_number],
-      qts: validated_record[:qts_date].present? && validated_record[:qts_date] != "null",
-      active_alert: validated_record[:active_alert],
-      previous_participation: false,
-      previous_induction: false,
+    {
+      trn: validated_record["trn"],
+      qts: validated_record.dig("qualified_teacher_status", "qts_date").present?,
+      active_alert: validated_record["active_alert"],
+      previous_participation: previous_participation?(validated_record),
+      previous_induction: previous_induction?(validated_record),
     }
-
-    set_eligibility_flags(validation_data)
   end
 
 private
 
-  def set_eligibility_flags(validation_data)
-    ineligibility_flag = CheckParticipantEligibility.call(trn: validation_data[:trn])
+  def previous_participation?(validation_data)
+    CheckParticipantPreviousParticipation.call(trn: validation_data["trn"])
+  end
 
-    if ineligibility_flag.present?
-      if ineligibility_flag == :previous_induction_and_participation
-        validation_data[:previous_participation] = validation_data[:previous_induction] = true
-      else
-        validation_data[ineligibility_flag] = true
-      end
-    end
+  def previous_induction?(validation_data)
+    return false if validation_data["induction"].nil?
+    return true if validation_data["induction"]["completion_date"].present?
 
-    validation_data
+    validation_data["induction"]["start_date"] < Date.new(Cohort.current.start_year, 9, 1)
   end
 
   def check_first_name_only?
     config[:check_first_name_only]
   end
 
   def dqt_record(trn, nino)
-    if FeatureFlag.active?(:full_dqt_api)
-      object = full_dqt_client.get_record(trn: trn, birthdate: date_of_birth, nino: nino)
-      FullDqt::OldRecordMapper.translate(object)
```

Comment:
Should we cut out `FullDqt::OldRecordMapper` too?

Reply:
Yep, good spot. Had intended to